[NUI] Enhance FlexLayout documents (#1392)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/flex-layout.md

flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
layoutView.Layout = flexLayout;
```
There are six properties that affect the specified child view.

> [!NOTE]

> > These properties are added in Tizen 6.0.

Reviewer: @dasg34
How about using a common note instead of individual notes? 

> [!NOTE]
> The `FlexAlignmentSelf`, `FlexPositionType` ... ... properties are added in Tizen 6.0.

We can remove individual notes.

Following are the properties:

| Property               | Type            | Description  |
| -----------------------| --------------- | ------------ |
| `FlexAlignmentSelf`    | AlignmentType   | The alignment of item along the cross axis when free space is available. |
| `FlexPositionType`     | PositionType    | The position type that defines how it is positioned within its parent. |
| `FlexAspectRatio`      | float           | The ratio between the width and the height of a item |
| `FlexBasis`            | float           | The axis-independent way of providing the default size of an item along the main axis. |
| `FlexShrink`           | float           | The Flex shrink describes how to shrink children along the main axis in the case that the total size of the children overflow the size of the container on the main axis. |

> | `FlexGrow`             | float           | The Flex grow describes how any space within a container should be distributed among its children along the main axis. |

Reviewer: Please rephrase this:
The axis-independent way of providing the item's default size along the main axis. |

> 

Reviewer: Following are the properties:

| Property               | Type            | Description  |
| -----------------------| --------------- | ------------ |
| `FlexAlignmentSelf`    | AlignmentType   | The alignment of item along the cross axis when free space is available. |
| `FlexPositionType`     | PositionType    | The position type that defines how it is positioned within its parent. |
| `FlexAspectRatio`      | float           | The ratio between the width and the height of a item |
| `FlexBasis`            | float           | The axis-independent way of providing the default size of an item along the main axis. |
| `FlexShrink`           | float           | The Flex shrink describes how to shrink children along the main axis in the case that the total size of the children overflow the size of the container on the main axis. |
| `FlexGrow`             | float           | The Flex grow describes how any space within a container should be distributed among its children along the main axis. |

> ## FlexAlignmentSelf property

Reviewer: @dasg34 please resolve the comment, sighting your opinion. 
Thanks.

> ## FlexAlignmentSelf property

Author: Done.

> [!NOTE]
> This property is added in Tizen 6.0.

The `FlexAlignmentSelf` has the same options and effect as `Alignment` but instead of affecting the children within a container, you can apply this property to a single child to change its alignment within its parent.
The align self overrides any option set by the parent with align items.

The following figure shows how the second item works with `FlexAlignmentSelf` property:

![FlexAlignmentSelf](./media/flexAlignmentSelf.png)

> 

Author: The second item is a rectangle with text "2" in the figure..

> 

Author: It changed to 
```
The following figure shows how to work the `FlexAlignmentSelf` property for item `2`: 
```

## FlexPositionType property

> [!NOTE]
> This property is added in Tizen 6.0.

The `FlexPositionType` property of an element defines how it is positioned within its parent.

By default an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and then offset relative to that margin based on the values of start, end, top, and bottom.
The offset does not affect the position of any sibling or parent elements.

When positioned absolutely an element doesn't take part in the normal layout flow.
It is instead laid out independent of its siblings. The position is determined based on the position property.

The following figure shows how the second item works with `FlexPositionType` property:

> ![FlexPositionType](./media/flexPositionType.png)

Author: It's changed to 
```
The following figure shows how to work the `FlexPositionType` property for item `2`:
```

## FlexAspectRatio property

> [!NOTE]
> This property is added in Tizen 6.0.

The `FlexAspectRatio` property has the following properties:

 - Accepts any floating point value > 0, the default is undefined.
 - Defined as the ratio between the width and the height of a node e.g. if a node has an aspect ratio of 2 then its width is twice the size of its height.
 - Respects the min and max dimensions of an item.
 - Has higher priority than `FlexGrow` property.

>  - If aspect ratio, width, and height are set then the cross axis dimension is overridden.

Author: This sentence is removed.

> ## FlexBasis property
> 
> > [!NOTE]
> > This property is added in Tizen 6.0.
> 
> The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
> Setting the flex basis of a child is similar to setting the width of that child if its parent is a container with flex direction: row or setting the height of a child if its parent is a container with flex direction: column.
> The flex basis of an item is the default size of that item, the size of the item before any flex grow and flex shrink calculations are performed.

Reviewer: ```suggestion
## FlexBasis property

The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
Setting the flex-basis of a child is similar to setting the width of that child if its parent is a container with:
-  `flexdirection: row`, or setting the height of a child if its parent is a container with `flex direction: column`.

The flex-basis of an item is the default size of that item, the size of the item before any flex-grow and flex-shrink calculations are performed.

> [!NOTE]
> This property is added in Tizen 6.0.

```

Not able to understand this, can you give more clarity?

Setting the flex basis of a child is similar to setting the width of that child if its parent is a container with flex direction: row or setting the height of a child if its parent is a container with flex direction: column.
```

Author: It's changed to
```
Setting the flex basis of a child is similar to setting the width of that child if its parent flex layout is set flex direction to `Row` or setting the height of a child if its parent flex layout set flex direction to `Column`.
```

> ## FlexShrink property
> 
> > [!NOTE]
> > This property is added in Tizen 6.0.
> 
> The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflow the size of the container on the main axis.
> The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. These two properties also work well together by allowing children to grow and shrink as needed.
> 
> The flex shrink accepts any floating point value >= 0, with 1 being the default value. A container will shrink its children weighted by the child’s flex shrink value.
> 
> The following figure shows how the second item works with `FlexShrink` property:
> 
> ![FlexShrink](./media/flexShrink.png)
> 

Reviewer: ```suggestion
## FlexShrink property

The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflows the size of the container on the main axis.
The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. These two properties also work well together by allowing children to grow and shrink as needed.

The flex shrink accepts any floating point value greater or equal to `0`. The default value is `1`. A container will shrink its children weighted by the child’s flex shrink value.
The following figure shows how the second item works with `FlexShrink` property:

![FlexShrink](./media/flexShrink.png)

> [!NOTE]
> This property is added in Tizen 6.0.

```
Can you please rephrase thes?

The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflows the size of the container on the main axis.

The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space. 

The flex shrink accepts any floating point value >= 0, with 1 being the default value. A container will shrink its children weighted by the child’s flex shrink value.

Author: > The flex shrink is very similar to flex grow and can be thought of in the same way if any overflowing size is considered to be negative remaining space.
> 
> A container will shrink its children weighted by the child’s flex shrink value.

Above sentences have been removed.

> ## FlexGrow property
> 
> > [!NOTE]
> > This property is added in Tizen 6.0.
> 
> The `FlexGrow` property describes how any space within a container should be distributed among its children along the main axis. After laying out its children, a container will distribute any remaining space according to the flex grow values specified by its children.
> 
> The flex grow accepts any floating point value >= 0, with 0 being the default value. A container will distribute any remaining space among its children weighted by the child’s flex grow value.
> 
> The following figure shows how the second item works with `FlexGrow` property:
> 
> ![FlexGrow](./media/flexGrow.png)

Reviewer: ```suggestion

## FlexGrow property

The `FlexGrow` property describes how the space within a container should be distributed among its children along the main axis. After laying out its children, a container distributes any remaining space according to the flex grow values specified by its children.

The flex grow accepts any floating point value greater or equal to `0`. The default value is `0`. A container will distribute any remaining space among its children weighted by the child’s flex grow value.

The following figure shows how the second item works with `FlexGrow` property:

![FlexGrow](./media/flexGrow.png)

> [!NOTE]
> This property is added in Tizen 6.0.
```

Please rephrase these: 
A container will distribute any remaining space among its children weighted by the child’s flex grow value.

The following figure shows how the second item works with `FlexGrow` property:

Author: > A container will distribute any remaining space among its children weighted by the child’s flex grow value.

This has been removed.

> The following figure shows how the second item works with `FlexGrow` property:

=>  It's changed to 
```
The following figure shows how to work the `FlexGrow` property for item `2`:
```

> ## Related Information

Reviewer: Please ensure to follow sentence case for all headings.

Author: Done.

flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
layoutView.Layout = flexLayout;
```
There are six properties that affect the specified child view.

> [!NOTE]
> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, `FlexGrow` properties are added in Tizen 6.0.

> The following properties affect the specified child view:

Reviewer: We can remove this line and add the following instead.
```suggestion 
**Table: Properties and descriptions**

| Property               | Type            | Description  |
| -----------------------| --------------- | ------------ |
| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |

> | `FlexPositionType`     | PositionType    | The position type defines how it is positioned within its parent. |

Author: "it" is an item:)

 - Defined as the ratio between the width and the height of a node, e.g. if a node has an aspect ratio of `2`, then its width is twice the size of its height.
 - Has a higher priority than `FlexGrow` property.
 - If aspect ratio, width, and height are set then the cross axis dimension is overridden.

## FlexBasis property

The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
Setting the flex basis of a child is similar to setting the width of that child if its parent flex layout is set flex direction to `Row` or setting the height of a child if its parent flex layout set flex direction to `Column`.

The flex basis of an item is the default size of that item, the size of the item before any flex grow and flex shrink calculations are performed.

## FlexShrink property

The `FlexShrink` describes how to shrink children along the main axis in the case that the total size of the children overflows the size of the flex layout on the main axis.

The flex shrink accepts any floating point value greater or equal to `0`. The default value is `1`.

> The following figure shows how to work the `FlexShrink` property for item `2`:

Reviewer: ```suggestion
The `FlexShrink` specifies how to shrink the children along the main axis in case, the total size of the children overflows the size of the flex layout on the main axis. The flex-shrink accepts any floating-point value greater or equal to `0`. The default value is `1`.

The following figure shows how to use the `FlexShrink` property:

![FlexShrink](./media/flexShrink.png)

## FlexGrow property

The `FlexGrow` property describes how any space within a flex layout should be distributed among its children along the main axis. After laying out its children, a flex layout distributes any remaining space according to the flex grow values specified by its children.

The flex grow accepts any floating point value greater or equal to `0`. The default value is `0`.

> The following figure shows how to work the `FlexGrow` property for item `2`:

Reviewer: ```suggestion
The `FlexGrow` property specifies how space within a flex layout should be distributed among its children along the main axis. After laying out its children, a flex layout distributes any remaining space according to the flex-grow values specified by its children. The flex-grow accepts any floating-point value greater or equal to `0`. The default value is `0`.

The following figure shows how to use the `FlexGrow` property:

layoutView.Layout = flexLayout;
```

 The following properties affect the specified child view areas:
- `FlexAlignmentSelf`
-  `FlexPositionType`
-  `FlexAspectRatio`
-  `FlexBasis`
-  `FlexShrink`
-  `FlexGrow`

> [!NOTE]
> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, and `FlexGrow` properties are added in Tizen 6.0.

**Table: Properties and descriptions**

| Property               | Type            | Description  |
| -----------------------| --------------- | ------------ |
| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |
| `FlexPositionType`     | PositionType    | The position type defines how it is positioned within its parent. |
| `FlexAspectRatio`      | float           | The ratio between the width and the height of an item. |

> | `FlexBasis`            | float           | The item's default size before any flex grow and flex shrink calculations are performed. |

Reviewer: ```suggestion
| `FlexBasis`            | float           | TSpecifies item's default size before `FlexGrow` and `FlexShrink` calculations are performed. |

> | `FlexShrink`           | float           | Specifies how to shrink children along the main axis in the case that the total size of the children overflows the size of the flex layout on the main axis. |

Reviewer: ```suggestion
| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in case, the total size of the children overflows the size of the flex layout on the main axis. |

| `FlexGrow`             | float           | Specifies how the space within a flex layout is distributed among its children along the main axis. |

> 

Reviewer: @mijin85cho yes I do agree.

 I had [suggested](https://github.com/Samsung/tizen-docs/pull/1392#discussion_r56068237)
then we had this [discussion](https://github.com/Samsung/tizen-docs/pull/1392#discussion_r558148869). After wards author removed this. 

Can you please suggest what will be the best fit here. 
Thanks.

> 

Reviewer: @dasg34 I leave it to you to take a call on this good suggestion.

> 

Reviewer 2: > @mijin85cho yes I do agree.
> 
> I had [suggested](https://github.com/Samsung/tizen-docs/pull/1392#discussion_r56068237)
> then we had this [discussion](https://github.com/Samsung/tizen-docs/pull/1392#discussion_r558148869). After wards author removed this.
> 
> Can you please suggest what will be the best fit here.
> Thanks.
> cc: @annie-samsung

@nawab-samsung I think you confused the comments, as the history you traced is in wrong order.

You said [you suggested](https://github.com/Samsung/tizen-docs/pull/1392#discussion_r560682379)(commented on 20 Jan) and then had [discussion](https://github.com/Samsung/tizen-docs/pull/1392#discussion_r558148869)(commented on 15 Jan). However as the comment of 15 Jan is previous one... Author updated the sentence, and you guided to remove "for item `2`". 

Anyway, as I said, I think the original one or the second one updated by author is good to apply.

> 

Author: I don't want to change this patch any more. if you find better one, feel free to replace it.

## FlexAlignmentSelf property

> The `FlexAlignmentSelf` property has the same options and effect as `Alignment` property, but instead of affecting the children within a flex layout, this property is applied to a single child to change its alignment within its parent. The align-self overrides any option set by the parent with align-items.

Reviewer: ```suggestion
The `FlexAlignmentSelf` property has the same options and effect as the `Alignment` property, but instead of affecting the children within a flex layout, this property is applied to a single child to change its alignment within its parent. The align-self overrides any option set by the parent with align-items.
```

If align-self,  align-items are properties we need to keep them in code style.

Author: Done.

The following figure shows how to use the `FlexAlignmentSelf` property:

![FlexAlignmentSelf](./media/flexAlignmentSelf.png)

## FlexPositionType property

> The `FlexPositionType` property of an element defines how it is positioned within its parent. By default, an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and moved depending on the margin.

Reviewer: ```suggestion
The `FlexPositionType` property of an element defines how it is positioned within its parent. By default, an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and its movement depends on the margin.

> The offset does not affect the position of any sibling or parent elements. When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independently of its siblings.

Reviewer: I had little doubt here

> The offset does not affect the position of any sibling or parent elements. When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independently of its siblings.

Author: "offset" is position set by [Position2D](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_Position2D).

The following figure shows how to use the `FlexPositionType` property:

![FlexPositionType](./media/flexPositionType.png)

## FlexAspectRatio property

> 

Author: Done.

> The `FlexAspectRatio` property has the following properties:

Reviewer: Can we say it like this. I might be wrong. Please correct me then. 
```suggestion
The `FlexAspectRatio` property has the following characteristics:
 - Accepts any floating-point value greater than `0`. The default value is undefined.
 - Defined as the ratio between the width and the height of a node, for example, if a node has an aspect ratio of `2`, then its width is twice the size of its height.
 - Has higher priority than `FlexGrow` property.
 - If aspect ratio, width, and height are set, then the cross axis dimension is overridden.

Reviewer: Please see all the comments. https://github.com/Samsung/tizen-docs/pull/1392/files#r580181205
https://github.com/Samsung/tizen-docs/pull/1392/files#r580134933

Author: Done.

 - Accepts any floating-point value greater than `0`. The default value is undefined.

>  - Defined as the ratio between the width and the height of a node, e.g. if a node has an aspect ratio of `2`, then its width is twice the size of its height.

Reviewer: ```suggestion 
 - Defined as the ratio between the width and the height of a node, for example,  if a node has an aspect ratio of `2`, then its width is twice the size of its height.
 ```

 - Has a higher priority than `FlexGrow` property.
 - If aspect ratio, width, and height are set, then the cross axis dimension is overridden.

## FlexBasis property

The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis.
Setting the flex basis of a child is similar to setting the width of that child if its parent flex layout is set flex direction to `Row` or setting the height of a child if its parent flex layout set flex direction to `Column`.

> The flex basis of an item is the default size of that item which means the measured size can be changed by the flex-grow and the flex-shrink.

Reviewer: Can you please rephrase this a little for better comprehension? 
Is flex-basis a property?

Author: Yes, flex-basis is a property.

> The `FlexBasis` property specifies the default size of the child, but the child-size can also be changed with the `FlexGrow` and the `FlexShrink` property.

Reviewer: ```suggestion
- If `FlexDirection` of flex layout is `Column`, the `FlexBasis` property of a child is similar to height of the child.

The `FlexBasis` property specifies the default size of the child, but the child-size can also be changed with the `FlexGrow` and the `FlexShrink` property.

> The `FlexBasis` property specifies the default size of the child, but the child-size can also be changed with the `FlexGrow` and the `FlexShrink` property.

Reviewer: ```suggestion
- If `FlexDirection` of flex layout is `Column`, the `FlexBasis` property of a child is similar to height of the child.

The `FlexBasis` property specifies the default size of the child, but the child-size can also be changed with the `FlexGrow` and the `FlexShrink` property.

> The `FlexBasis` property specifies the default size of the child, but the child-size can also be changed with the `FlexGrow` and the `FlexShrink` property.

Reviewer: This is important to have a separation between content
@dasg34 Please do look at the suggestion.

> The `FlexBasis` property specifies the default size of the child, but the child-size can also be changed with the `FlexGrow` and the `FlexShrink` property.

Author: Done.

> The `FlexBasis` property specifies the default size of that item which means the measured size can be changed by the `FlexGrow` property and the `FlexShrink` property.

Reviewer: This is quite hard to comprehend, can we have a rephrase for these sentences.

## FlexShrink property

> The `FlexShrink` describes how to shrink children along the main axis in the case that total size of the children overflows the size of the flex layout on the main axis. The flex-shrink accepts any floating-point value greater or equal to `0`. The default value is `1`.

Reviewer: ```suggestion 
The `FlexShrink` describes how to shrink children along the main axis in case that the total size of the children overflows the size of the flex layout on the main axis. The `FlexShrink` accepts any floating-point value greater or equal to `0`. The default value is `1`.
```
Assuming flex shrink is a property

> The `FlexShrink` describes how to shrink children along the main axis in case that the total size of the children overflows the size of the flex layout on the main axis. The `FlexShrink` property accepts any floating-point value greater or equal to `0`. The default value is `1`.

Reviewer: ```suggestion
The `FlexShrink` describes how to shrink children along the main axis in case the total size of the children overflows the size of the flex layout on the main axis. The `FlexShrink` property accepts any floating-point value greater or equal to `0`. The default value is `1`.

The following figure shows how to use the `FlexShrink` property:

![FlexShrink](./media/flexShrink.png)

## FlexGrow property

> The `FlexGrow` property specifies how space within a flex layout should be distributed among its children along the main axis. After laying out its children, a flex layout distributes any remaining space according to the flex-grow values specified by its children. The flex grow accepts any floating-point value greater or equal to `0`. The default value is `0`.

Reviewer: ```suggestion
The `FlexGrow` property specifies how the space within a flex layout is distributed among its children along the main axis. After laying out its children, a flex layout distributes any remaining space according to the `FlexGrow` values specified by its children. The `FlexGrow` accepts any floating-point value greater or equal to `0`. The default value is `0`.

| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in case the total size of the children overflows the size of the flex layout on the main axis. |
| `FlexGrow`             | float           | Specifies how the space within a flex layout is distributed among its children along the main axis. |

## FlexAlignmentSelf property

The `FlexAlignmentSelf` property has the same options and effect as the `Alignment` property, but instead of affecting the children within a flex layout, this property is applied to a single child to change its alignment within its parent. The `FlexAlignmentSelf` property overrides any option set by the parent with the `Alignment` property.

The following figure shows how to use the `FlexAlignmentSelf` property:

![FlexAlignmentSelf](./media/flexAlignmentSelf.png)

## FlexPositionType property

The `FlexPositionType` property of an element defines how it is positioned within its parent. By default, an element is positioned relatively. This means an element is positioned according to the normal flow of the layout, and its movement depends on the margin.

> The offset does not affect the position of any sibling or parent elements. When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independently of its siblings. The position is determined based on the position property.

Reviewer: The position is determined based on the position property.

Should position before property be in code style? 

What do you say, as this is property?

> The offset does not affect the position of any sibling or parent elements. When positioned absolutely, an element doesn't take part in the normal layout flow. It is instead laid out independently of its siblings. The position is determined based on the position property.

Author: I will remove this ambiguous sentence.

The following figure shows how to use the `FlexPositionType` property:

![FlexPositionType](./media/flexPositionType.png)

## FlexAspectRatio property

The `FlexAspectRatio` property has the following properties:

 - Accepts any floating-point value greater than `0`. The default value is undefined.
 - Defined as the ratio between the width and the height of a node, for example, if a node has an aspect ratio of `2`, then its width is twice the size of its height.
 - Has a higher priority than `FlexGrow` property.
 - If aspect ratio, width, and height are set, then the cross axis dimension is overridden.

## FlexBasis property

The `FlexBasis` property is an axis-independent way of providing the default size of an item along the main axis:
 - If `FlexDirection` of flex layout is `Row`, the `FlexBasis` property of a child is similar to width of the child.
 - If `FlexDirection` of flex layout is `Column`, the `FlexBasis` property of a child is similar to height of the child.

> 

Reviewer: Please remove this extra line space.

> The `FlexBasis` property specifies the default size of the child. but the child size can be changed depending on the `FlexGrow` property and the `FlexShrink` property.

Author: It is better :)

- `FlexBasis`
- `FlexShrink`
- `FlexGrow`

> [!NOTE]
> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, and `FlexGrow` properties are added in Tizen 6.0.

**Table: Properties and descriptions**

| Property               | Type            | Description  |
| -----------------------| --------------- | ------------ |
| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |
| `FlexPositionType`     | PositionType    | The position type defines how the item is positioned within its parent. |
| `FlexAspectRatio`      | float           | The ratio between the width and the height of an item. |
| `FlexBasis`            | float           | Specifies item's default size before `FlexGrow` and `FlexShrink` calculations are performed. |
| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in case the total size of the children overflows the size of the flex layout on the main axis. |
| `FlexGrow`             | float           | Specifies how the space within a flex layout is distributed among its children along the main axis. |

> ### FlexAlignmentSelf property

Reviewer 3: After the second level heading in line 93 ## FlexWrapType, the next heading is level 3
The staging link is not working, so I am unable to confirm how the right-side TOC will appear.

Is the intention that all the third-level headings are to be grouped under FlexWrapType? : 
 ## FlexWrapType
### FlexAlignmentSelf property
### FlexPositionType property
### FlexAspectRatio property
### FlexBasis property
### FlexShrink property
### FlexGrow property

If not can we have a meaningful second-level heading under which these third-level headings can be grouped?

Author: second-level heading `Attached Property` is added.

Reviewer 3: @dags34, thanks for fixing.
